Client/Frontend/Browser/New Tab Page/Sections/FavoritesSectionProvider.swift

```diff
@@ -30,7 +30,7 @@ class FavoritesSectionProvider: NSObject, NTPObservableSectionProvider {
         
         frc = Bookmark.frc(forFavorites: true, parentFolder: nil)
         super.init()
-        frc.fetchRequest.fetchLimit = 6
+        frc.fetchRequest.fetchLimit = 20
```

Comment:
I wonder if we should refetch the data after favorite update/delete.
Not a big deal, since user would have to remove 15 favs in a row to go over this fetchLimit

Reply:
Update: I added `performFetch` it makes sense to have it